src/mmap_unix.rs

```diff
@@ -172,6 +173,7 @@ impl MmapRegion {
             prot,
             flags,
             owned: true,
+            hugetlbfs: false,
```

Comment:
Can we have the `hugetlbfs` field of `MmapRegion` as an `Option<bool>` which is `None` initially? (since there's no available information right now, until we explicitly set it with `set_hugetlbfs`)